### PR TITLE
don't sleep in TestSystemMonitor.test_login_logout_messages

### LINKED ISSUE #2721: sleeping in TestSystemMonitor.test_login_logout_messages is not a solution
```
self = <tests.golem.monitor.test_monitor.TestSystemMonitor testMethod=test_login_logout_messages>

    def test_login_logout_messages(self):
        """Test whether correct login and logout messages
                and protocol data were sent."""
    
        def check(f, msg_type):
            with mock.patch(
                'golem.monitor.transport.httptransport.'
                + 'DefaultHttpSender.post_json') \
                    as mock_send:
                f()
                time.sleep(0.005)
                self.assertEqual(mock_send.call_count, 1)
                result = json.loads(mock_send.call_args[0][0])
                expected_d = {
                    'proto_ver': MONITOR_CONFIG['PROTO_VERSION'],
                    'data': {
                        'type': msg_type,
                        'protocol_versions': {
                            'monitor': self.monitor.config['PROTO_VERSION'],
                            'p2p': variables.PROTOCOL_CONST.ID,
                            'task': variables.PROTOCOL_CONST.ID,
                        },
                        'metadata': {
                            'type': 'NodeMetadata',
                            'net': 'testnet',
                            'timestamp': mock.ANY,
                            'cliid': 'cliid',
                            'sessid': 'sessid',
                            'os': 'os',
                            'version': 'ver',
                            'settings': mock.ANY,
                        },
                        'cliid': 'cliid',
                        'sessid': 'sessid',
                        'timestamp': mock.ANY,
                    }
                }
                self.assertEqual(expected_d, result)
    
>       check(self.monitor.on_login, "Login")

tests/golem/monitor/test_monitor.py:112: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/golem/monitor/test_monitor.py:84: in check
    self.assertEqual(mock_send.call_count, 1)
E   AssertionError: 0 != 1
```
While it works on a developer's PCs, it sometimes fails on external CI.

### PR DESCRIPTION
**Pull request by etam**
Fixes #2721

### CODE REVIEW

File: tests/golem/monitor/test_monitor.py [hunk 1]
```diff
@@ -37,10 +37,10 @@ def setUp(self):
         config['HOST'] = 'http://localhost/88881'
         config['SENDER_THREAD_TIMEOUT'] = 0.05
         self.monitor = SystemMonitor(meta_data, config)
-        self.monitor.start()
+        # Don't start sending thread here.
```
Comment (Krigpl) [inline tests/golem/monitor/test_monitor.py]: How about making it a proper unit test by mocking out the `SenderThread` and checking whether `send` was called with proper args?

**Comment by Krigpl** [comment]
catping

**Comment by etam** [comment]
@Krigpl 
pong

**Review by Krigpl — APPROVED** [review]
ok_thumb

**Pull request merged**